There's a TSC error:
```
src/y-socket-io/y-socket-io.js:475:50 - error TS2345: Argument of type 'Awareness | null' is not assignable to parameter of type 'Awareness'.
  Type 'null' is not assignable to type 'Awareness'.

475           AwarenessProtocol.applyAwarenessUpdate(existDoc.awareness, msg, null)
                                                     ~~~~~~~~~~~~~~~~~~


Found 1 error in src/y-socket-io/y-socket-io.js:475
```

https://github.com/hackmdio/y-socketio-redis/blob/a13c93e89a4fe9abd1fc07a02b98d9cebb661faf/src/y-socket-io/y-socket-io.js#L473-L476

Should be:
```ts
        if (existDoc.awareness) {
          for (const msg of awareness) {
            if (msg.length === 0) continue
            AwarenessProtocol.applyAwarenessUpdate(existDoc.awareness, msg, null)
          }
        }
```

--- a/src/y-socket-io/y-socket-io.js
+++ b/src/y-socket-io/y-socket-io.js
@@ -470,9 +470,11 @@ export class YSocketIO {
           if (msg.length === 0) continue
           Y.applyUpdate(existDoc.ydoc, msg)
         }
-        for (const msg of awareness) {
-          if (msg.length === 0) continue
-          AwarenessProtocol.applyAwarenessUpdate(existDoc.awareness, msg, null)
+        if (existDoc.awareness) {
+          for (const msg of awareness) {
+            if (msg.length === 0) continue
+            AwarenessProtocol.applyAwarenessUpdate(existDoc.awareness, msg, null)
+          }
         }
       })
     }

